pkg/e2e/compose_test.go

```diff
@@ -125,8 +125,28 @@ func TestComposePull(t *testing.T) {
 	res := c.RunDockerComposeCmd(t, "--project-directory", "fixtures/simple-composefile", "pull")
 	output := res.Combined()
 
-	assert.Assert(t, strings.Contains(output, "simple Pulled"))
-	assert.Assert(t, strings.Contains(output, "another Pulled"))
+	expected := []string{
+		"Skipped - No image to be pulled",
+		"Skipped - Image is already present locally",
+		"Skipped - Image is already being pulled by",
+		"simple Pulled",
+		"another Pulled",
+	}
+
+	assert.Assert(t, contains(output, expected))
+}
+
+func contains(str string, array []string) bool {
```

Comment:
nit: `contains()` could be a bit cleaner:
```
func contains(str string, array []string) bool {
	for _, val := range array {
		if strings.Contains(str, val) {
			return true
		}
	}
	return false
}
```

Reply:
after updating test this is no longer required